src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import core.basesyntax.reader.CsvFileReader;
+import core.basesyntax.report.CsvFileWriter;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
The `main()` method should be responsible for orchestrating the flow of the application. This includes reading the file, processing the data, and writing the report. Each of these steps should be performed by calling independent services in the right order.